code/controllers/subsystem/ticker.dm

```diff
@@ -322,7 +322,7 @@ SUBSYSTEM_DEF(ticker)
 			if(M.stat != DEAD)
 				var/turf/T = get_turf(M)
 				if(T && is_station_level(T.z) && !istype(M.loc, /obj/structure/closet/secure_closet/freezer) && !(issilicon(M) && override == "AI malfunction"))
-					to_chat(M, "<span class='danger'><B>The blast wave from [src] tears you atom from atom!</B></span>")
```

Comment:
And your PR title had me thinking there were major flaws in SSticker.


You're fired.